javascript-modal/main.js

```diff
@@ -0,0 +1,24 @@
+var $modal = document.querySelector('.invisible');
+var $modalButton = document.querySelector('.open-modal');
+var $overlay = document.querySelector('.no-overlay');
+var $redButton = document.querySelector('.no-button');
+var isModalOpen = false;
+
+function modalHandler(event) {
+  if (isModalOpen === false) {
+    $modal.className = 'visible';
+    $overlay.className = 'overlay';
+    isModalOpen = true;
+  }
+}
+
+function modalHandlerTwo(event) {
+  if (isModalOpen === true) {
```

Comment:
This conditional is unnecessary. Any time this function runs, you want the modal to close.